src/Components/books.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const Books = () => (
+  <>
+    <h2>Hello Books</h2>
+    <li>The new Horizon By david Norson</li>
+    <li>The lands of Arcanum By rida Bensalem</li>
+    <li> The pain By Ahmed sefroui</li>
+    <li>The last day by Victor Hugo</li>
+    <br />
+    <form action="post ">
+      Book Name :
+      {' '}
+      <input type="text" />
+      {' '}
+      <br />
+      {' '}
+      <br />
+      Book Author :
+      {' '}
+      <input type="text" />
+      {' '}
+      <br />
+      {' '}
+      <br />
+      <button type="submit"> add a book</button>
+      {' '}
+      <br />
+    </form>
+  </>
+);
```

Comment:
According to the requirement, you are to create a  component called Book for displaying a single book (it will receive a title and an author as prop) and reuse it in a component that displays a list of books(BookList). 😊